doc/docportal/advanced_topics/understand_graphics.rst

```diff
@@ -20,8 +20,6 @@ Graphics modes
 
 Most platforms have either one or two available graphics modes. The most common ones are OpenGL and SDL Surface.
 
-To switch between graphics modes, press :kbd:`Ctrl + Alt` and :kbd:`1` to :kbd:`8`.
```

Comment:
An amended version of this could maybe move a bit further down the document, to the Comparison of SDL Software Scalers section? Something like "To cycle forwards or backwards between graphical filters, press :kbd:`Ctrl + Alt` and 0 or 9." just before the example images

Reply:
Done in a new commit. However, some more updating could be required (maybe @criezy would know better) because that section may have to be renamed (to include OpenGL mode as well) and also a bit above where we mention that "For the OpenGL graphics mode there is no scaler". I think those need to be changed but not sure what to change them with.

Reply:
Indeed more changes are needed. This page was written at a time where the software scalers were not available with the OpenGL graphics mode. But now that they are available:
* The description of the `OpenGL graphics mode` section is incorrect.
* The `A comparison of SDL Surface software scalers` subsection should be renamed and move out of the `SDL Surface graphics mode` section.

The main difference that remains between `OpenGL` and `SDL Surface` is for the aspect ratio correction that is done at the same time as applying the stretch mode for OpenGL, while the two are done in two separate passes for SDL Surface. And there is one important new difference: we now have shaders available in OpenGL mode.

I am not quite sure how shaders interact with the stretch mode, but my understanding is that some of them can do a scaling. Does the stretch mode then apply to the result of that scaling? Are some shaders applied before stretch and others after? That could be a point interesting to clarify in the documentation, but we would need information from a somebody more familiar with how shaders are implemented, because I don't have the answer myself.

Reply:
Do you think that these updates should be part of this PR? I'd say I agree that it's a good opportunity to update the rest of the content since it's outdated, but I lack the knowledge on what exactly to update myself.

Reply:
I'd be happy to work on a more comprehensive update for the page (although probably only next week or the week after). Possibly easier if I start a new PR?

Reply:
Then let me merge this, so then @cadihowley could work on top of it.